kubernetes/istio.go

```diff
@@ -318,9 +322,9 @@ func parseRegistryServices(registries map[string][]byte) ([]*RegistryStatus, err
 
 func (in *K8SClient) GetConfigDump(namespace, podName string) (*ConfigDump, error) {
 	// Fetching the config_dump data, raw.
-	resp, err := in.EnvoyForward(namespace, podName, "/config_dump")
+	resp, err := in.ForwardGetRequest(namespace, podName, httputil.Pool.GetFreePort(), 15000, "/config_dump")
```

Comment:
I'd add a similar comment as used for port 15014, with the links of the ports used.